src/main/pages/setup-kubernetes/kubernetes-single-user.adoc

```diff
@@ -8,24 +8,21 @@ folder: setup-kubernetes
 ---
 
 [id="supported-kuberentes-flavors-and-versions"]
-== Supported Kuberentes Flavors and Versions
+== Supported Kuberentes flavors and versions
 
-Tested only on MiniKube v0.25 with vm providers VirtualBox and kvm2.
+The single-user deployment to Kubernetes is tested only on MiniKube v0.25 with the *VirtualBox* and *kvm2* VM providers.
```

Comment:
"MiniKube" should be "Minikube". This applies globally (I see three instances of "MiniKube" being used in the diff here).

Also, Minikube uses `docker-machine` driver plug-ins to provide functionality for hypervisors. I'd recommend changing "providers" to "drivers" or "driver plug-ins".